### PR TITLE
Pull to refresh & dark theme fixes

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: src/store/addressesSlice.ts [hunk 1]
```diff
@@ -105,18 +104,9 @@ export const fetchAddressesDataNextPage = createAsyncThunk(
...
-      let newTransactions = [] as Transaction[]
-
-      if (!allPagesLoaded) {
-        nextPage += 1
-        console.log(`⬇️ Fetching page ${nextPage} of address confirmed transactions: `, addressHash)
-        const { data: transactions } = await client.explorerClient.getAddressTransactions(addressHash, nextPage)
-        newTransactions = transactions
-      }
+      const page = 1
+      console.log(`⬇️ Fetching page ${page} of address confirmed transactions: `, addressHash)
```
Comment (mvaivre) [inline src/store/addressesSlice.ts:108]: Should we differentiate the log message from the one from `fetchAddressesDataNextPage`?
We fetch more than just the transactions here.
Also, if `page` is a constant, let's just remove it?

File: src/utils/transactions.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { Transaction } from '@alephium/sdk/api/explorer'
+
+export const getNewTransactions = (
```
Comment (mvaivre) [inline src/utils/transactions.ts:21]: `extractNewTransactions`? Just to differentiate it from an endpoint call.

**Review by mvaivre — COMMENTED** [review]
👌

File: src/utils/transactions.ts (hunk 2 above)
Comment (nop33) [reply src/utils/transactions.ts:21]: Yep, sounds appropriate!

File: src/store/addressesSlice.ts (hunk 1 above)
Comment (nop33) [reply src/store/addressesSlice.ts:108]: Yes, I'll rework it a bit, thanks!

File: src/utils/transactions.ts (hunk 2 above)
Comment (nop33) [reply src/utils/transactions.ts:21]: I included the fix in the next PR, I hope that's fine 😇 efd336e

File: src/store/addressesSlice.ts (hunk 1 above)
Comment (nop33) [reply src/store/addressesSlice.ts:108]: Same for this fix :) a0a2b3f

File: src/utils/transactions.ts (hunk 2 above)
Comment (mvaivre) [reply src/utils/transactions.ts:21]: Perfect :)

**Review by mvaivre — APPROVED** [review]

**Pull request merged**